docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -108,6 +108,20 @@ permission to restart the Nginx service as root. In Teleport 16.4.0 and later,
 the default shell for a created user can be configured with `create_host_user_default_shell`. 
 Otherwise the host's default shell will be used.
 
+<Admonition type="note">
+
+When using the terraform provider, the values for `create_host_user_mode` are represented numerically
```

Comment:
Since this guide assumes that you are using `tctl`, I don't think we need this Admonition. Instead, we can indicate in the "Next steps" section that you can manage host user creation using Terraform and link to the relevant Terraform resource reference guide.

Reply:
Would it be worth specifically calling out in "Next steps" that some configuration values might be different in the terraform provider than they are in this guide? This is mostly to try and reduce the potential confusion highlighted in #33288

Reply:
That makes sense to me!

Reply:
Added a link to the provider reference in next steps with a note about values being different :+1: